core/primitives/src/version.rs

```diff
@@ -147,6 +147,8 @@ pub enum ProtocolFeature {
     Ed25519Verify,
     #[cfg(feature = "protocol_feature_reject_blocks_with_outdated_protocol_version")]
     RejectBlocksWithOutdatedProtocolVersions,
+    #[cfg(feature = "protocol_feature_zero_balance_account")]
```

Comment:
let's add a comment and mention the NEP number.

I also wonder if we shouldn't put NEP number in the name of the field itself.. ZeroBalanceAccount_NEP448